packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -174,18 +175,16 @@ export function TelemetryView(): React.ReactElement {
 			/>
 			<div className={styles.menu}>
 				<div>
-					<div style={{ marginLeft: "6px" }}>
```

Comment:
You lost the outer div with margin, just pointing in case it was unintentional

Reply:
It was intentional. I added padding to the whole "content panel" so no individual view has to be concerned with handling its margins to create empty space near the edges.